ansible/vars/all.yml

```diff
@@ -19,3 +19,5 @@ pbench_configuration_environment: ec2
 pbench_key_url:
 pbench_config_url:
 pbench_record_duration: 360
+
+microshift_hostname: "{{ microshift_hostname_arg | default('microshift-dev') }}"
```

Comment:
We don't want to hardcode the hostname in various places or have to pass it as an argument.
The single source of truth should be the inventory which defines the hostnames etc:
```
jumper ansible_port=5555 ansible_host=192.0.2.50
```

There's a few ansible built-in variables that give us the hostname.
- `inventory_hostname` would be the hostname as defined in the inventory (`jumper` above)
- `ansible_hostname` would be the hostname that ansible discovers once it connects to the host